packages/atomic/src/components/common/facets/facet-common.tsx

```diff
@@ -620,8 +621,8 @@ export class FacetCommon {
     if (hasError || !this.facet.state.enabled) {
       return <Hidden></Hidden>;
     }
-
-    if (!firstSearchExecuted) {
+    console.log({shouldRender});
```

Comment:
Did you forget this or do we keep this to test until we have a complete solution ?

Reply:
I can take it out 👍